### PR TITLE
Add Bottom Panel Example

### PR DESCRIPTION
**Pull request by sundaram123krishnan**
So, i attempted to implement a bottom panel by referring through other examples.
Not sure how to make the window transparent further more i tried to make the `widget::container` transparent, but it didn't work.

### CODE REVIEW

**Comment by Decodetalkers** [comment]
Maybe you can ask iced for help... transparent will works on origin iced, right? but starcolorkeyboard can make it transparent.. so I think there is something not done in iced-layershell.. 

by the way. can you show a picture for me? about this example? and seems it failed in fmt

File: iced_examples/bottom_panel/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+license.workspace = true
+repository.workspace = true
+description.workspace = true
+keywords.workspace = true
+readme.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+iced = { version = "0.12", features = [
```
Comment (Decodetalkers) [inline iced_examples/bottom_panel/Cargo.toml:15]: use iced.workspace, since here do not use tokio

**Comment by Decodetalkers** [comment]
Can you add a Readme and picture for this example?

**Comment by sundaram123krishnan** [comment]
I think this is an issue with `iced-layershell`, in iced we have `transparent` field in the window `Settings`.
So, i suppose we need to add `transparent` field to the `LayerSettings`.

**Comment by Decodetalkers** [comment]
> I think this is an issue with `iced-layershell`, in iced we have `transparent` field in the window `Settings`.
> So, i suppose we need to add `transparent` field to the `LayerSettings`.

Yes, please help to add it

**Comment by sundaram123krishnan** [comment]
Yaa i can try, maybe in a separate pr? 
I think i need to modify the `draw` method in the lib?

**Comment by Decodetalkers** [comment]
> Yaa i can try, maybe in a separate pr? 
> I think i need to modify the `draw` method in the lib?

Sure!! But now I think this pr needs a README.

**Comment by sundaram123krishnan** [comment]
image
The panel looks smthg like this

File: iced_examples/bottom_panel/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        layer_settings: LayerShellSettings {
+            size: Some((600, 50)),
+            anchor: Anchor::Bottom,
+            layer: Layer::Overlay,
+            margin: (0, 0, 10, 0),
+            ..Default::default()
+        },
+        ..Default::default()
+    })
+    .unwrap();
```
[the remark this answers is not in the document]
Comment (sundaram123krishnan) [reply iced_examples/bottom_panel/src/main.rs:21]: Fine

File: iced_examples/bottom_panel/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+use applications::{all_apps, App};
+use iced::widget::row;
+use iced::{Command, Element, Theme};
+
+use iced_layershell::reexport::{Anchor, Layer};
+use iced_layershell::settings::{LayerShellSettings, Settings};
+use iced_layershell::Application;
+
+fn main() -> Result<(), iced_layershell::Error> {
+    let _ = Panel::run(Settings {
```
Comment (Decodetalkers) [inline iced_examples/bottom_panel/src/main.rs:11]:
```suggestion
fn main() -> Result<(), iced_layershell::Error> {
    Panel::run(Settings {
        layer_settings: LayerShellSettings {
            size: Some((600, 50)),
            anchor: Anchor::Bottom,
            layer: Layer::Overlay,
            margin: (0, 0, 10, 0),
            ..Default::default()
        },
        ..Default::default()
    })
}
```

**Comment by sundaram123krishnan** [comment]
Updated readme

File: iced_examples/bottom_panel/src/main.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (sundaram123krishnan) [reply iced_examples/bottom_panel/src/main.rs:11]: It gives warning, is it fine then?

**Comment by Decodetalkers** [comment]
Emm. I mean add a readme in `iced_examples/bottom_panel/README.md`

File: iced_examples/bottom_panel/src/main.rs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+use applications::{all_apps, App};
+use iced::widget::row;
+use iced::{Command, Element, Theme};
+
+use iced_layershell::reexport::{Anchor, Layer};
+use iced_layershell::settings::{LayerShellSettings, Settings};
+use iced_layershell::Application;
+
+fn main() -> Result<(), iced_layershell::Error> {
+    Panel::run(Settings {
```
Comment (Decodetalkers) [inline iced_examples/bottom_panel/src/main.rs:11]: I mean Panel::run already return `Result<(), iced_layershell::Error>`

look at this

https://github.com/waycrate/exwlshelleventloop/blob/master/iced_examples/counter/src/main.rs#L16-L25

**Comment by sundaram123krishnan** [comment]
Ok will add a separate `readme.md` for that, but in the main readme also let the image be present ?

**Comment by Decodetalkers** [comment]
> Ok will add a separate `readme.md` for that, but in the main readme also let the image be present ?

Yes. Image is better, but I think the image put in `iced_examples/bottom_panel` is better

File: iced_examples/bottom_panel/src/main.rs (hunk 4 above)
Comment (sundaram123krishnan) [reply iced_examples/bottom_panel/src/main.rs:11]: Yes yes got it, my bad

**Comment by Decodetalkers** [comment]
Emm. Maybe this example is good enough, ok, I will merge it

**Pull request merged**